### DISCUSSION TITLE
Server administration documents for Version 8.0.1

### DISCUSSION THREAD

**Discussion by lampok** [keycloak/keycloak#14527] [Q&A] [answered]
I am working to integrate an external OIDC provider into Keycloak. The software vendor we are using deployed version 8.0.1, I know old.

Does anyone have offline documentation for this version of Keycloak? Specifically looking for Server Administration docs for 8.0.1.

We are finding the differences between this version and the current version (even from version 12) are large enough that large gaps exist.

Ex. how do we create mappers to bring in data from the OIDC provider to skip the required data page.

**Comment by sschu** [comment, marked as answer]
@lampok Maybe you could just go back to an old commit of the keycloak-documentation repo like here: https://github.com/keycloak/keycloak-documentation/tree/3f6e60ccfa79971eb3ab3addee583ab713130c9e ?

**Reply by lampok** [reply]
This is perfect, I see 8.0.1 is in the tree. I will get on this right away

**Reply by lampok** [reply]
Unfortunately, there are missing files in the Repo for versions 7 and 8 to rebuild.

**Reply by sschu** [reply]
That's unfortunate. But you can probably still extract the important stuff for you from the source code...

**Reply by lampok** [reply]
Yes, I went rooting through the LDP files and extracted the text I needed. Thank you for your help.

**Reply by sschu** [reply]
@lampok  Glad to help. Can you please close this question as answered?
